JotunnLib/Extensions/MockExtentions.cs

```diff
@@ -84,7 +84,7 @@ public static Object GetRealPrefabFromMock(Object unityObject, Type mockObjectTy
                     // Cut off the suffix in the name to correctly query the original material
                     if (unityObject is Material)
                     {
-                        const string materialInstance = " (instance)";
+                        const string materialInstance = " (Instance)";
```

Comment:
better to be compared case insensitive.

Reply:
Not in the case where this is running recursive. The instance string is automated by unities clone process. The string is never not " Instance". All we are doing here is wasting cpu cycles in a O(n!) task. Lets not do that. Its not like I didn't consider this.

Reply:
fair, approved